packages/amplify-graphql-api-construct-tests/src/__tests__/restricted-field-auth/restricted-field-auth-gen1.test.ts

```diff
@@ -636,8 +636,8 @@ describe('Associated type fields with more restrictive auth rules than the model
 
       const leftRightJoin = result.body.data.createLeftRightJoin;
       expect(leftRightJoin).toBeDefined();
-      expect(leftRightJoin.manyLeft.secret).toBeNull();
```

Comment:
This is changing because these fields were previously required so they were not redacted before.